web/src/settings_notifications.ts

```diff
@@ -149,16 +173,19 @@ function stream_notification_setting_changed(e) {
     );
 }
 
-export function set_up(settings_panel) {
+export function set_up(settings_panel: SettingsPanel): void {
     const $container = $(settings_panel.container);
     const settings_object = settings_panel.settings_object;
-    const $notification_sound_elem = $(settings_panel.notification_sound_elem);
+    assert(settings_panel.notification_sound_elem !== null);
+    const $notification_sound_elem = $<HTMLVideoElement>(settings_panel.notification_sound_elem);
```

Comment:
Is this a video element? Probably deserves a comment, since it's definitely an element for a sound file, if there's some reason we have to use the video type here.

Reply:
Oh oops, I can just add `HTMLAudioElement` to the definition of `play_audio`